image/templates/helm/stackrox-secured-cluster/templates/_init.tpl.htpl

```diff
@@ -348,7 +348,6 @@ The following block checks for the validity of the provided init bundle. (`helm
   {{ if $centralDeployment.result }}
     {{ include "srox.note" (list $ "Detected central running in the same namespace. Not deploying scanner from this chart and configuring sensor to use existing scanner instance, if any.") }}
     {{ $_ := set $._rox.scanner "disable" true }}
-    {{ $_ := set $._rox.scannerV4 "disable" true }}
```

Comment:
This should not be necessary here due to the same check being made in the scanner v4 init logic:

https://github.com/stackrox/stackrox/blob/1220e226f6c2fe2b77c1d9f8eea299f444cd4cfe/image/templates/helm/shared/templates/_scanner-v4_init.tpl.htpl#L143-L146

This will ensure Scanner V4 isn't forcefully disabled in same namespace installs.